views/teams/edit-team.hbs

```diff
@@ -19,16 +19,14 @@
                     <input type="hidden" name="_method" data-force-value="true" value="{{method}}"/>
                 {{/if}}
 
-                <h4>Allgemeine Informationen</h4>
-
                 <div class="form-group">
                     <label>Name des Teams</label>
                     <input class="form-control" name="name" type="text" placeholder="A-Team" value="{{team.name}}"
                            required/>
                 </div>
 
                 <div class="form-group">
-                    <label>Beschreibung des Teams</label>
+                    <label>Beschreibung des Teams (optional)</label>
```

Comment:
In other cases we use `*` for required fields...

Reply:
I would slowly migrate this behaviour to (optional) + required fields with no additional marking. As of several reasons, check out the medium post if you want to dive deeper: 
https://uxdesign.cc/form-field-required-vs-optional-9b4d7cdbf400